_posts/2018-02-12-continuous-releases.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+tags: development-process
```

Comment:
Due to an annoying limitation of Jekyll on Github Pages, our tags are not entirely freeform and new tags need to be added in https://github.com/HypothesisWorks/HypothesisWorks.github.io/tree/master/articles

(there should probably be a script for this)